src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+    private double hypotenuse;
+
+    public RightTriangle(String color,
+                         double firstLeg,
+                         double secondLeg,
+                         double hypotenuse) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.hypotenuse = hypotenuse;
+    }
+
+    public double getFirstLeg() {
+        return this.firstLeg;
+    }
+
+    public void setFirstLeg(double firstLeg) {
+        this.firstLeg = firstLeg;
+    }
+
+    public double getSecondLeg() {
+        return this.secondLeg;
+    }
+
+    public void setSecondLeg(double secondLeg) {
+        this.secondLeg = secondLeg;
+    }
+
+    public double getHypotenuse() {
+        return this.hypotenuse;
+    }
+
+    public void setHypotenuse(double hypotenuse) {
+        this.hypotenuse = hypotenuse;
+    }
+
+    @Override
+    public double getArea() {
+        return this.firstLeg * this.secondLeg / 2;
```

Comment:
To ensure the encapsulation principle, it's better to use the getters for firstLeg and secondLeg instead of directly accessing the fields within the class.